WooCommerce/Classes/ViewRelated/Products/Edit Product/Product Settings/ProductSettingsRows.swift

```diff
@@ -350,6 +363,16 @@ extension ProductSettingsRows {
         static let catalogVisibility = NSLocalizedString("Catalog Visibility", comment: "Catalog Visibility label in Product Settings")
         static let virtualProduct = NSLocalizedString("Virtual Product", comment: "Virtual Product label in Product Settings")
         static let downloadableProduct = NSLocalizedString("Downloadable Product", comment: "Downloadable Product label in Product Settings")
+        static let downloadableProductAlertTitle = NSLocalizedString(
+            "Are you sure you want to remove the ability to download files when product is purchased?",
+            comment: "Confirm the change to make the product non downloadable"
+        )
+        static let downloadableProductAlertHint = NSLocalizedString(
+            "All files currently attached to this product will be removed.",
+            comment: "Confirm the change to make the product non downloadable"
+        )
+        static let downloadableProductAlertConfirm = NSLocalizedString("Yes, change", comment: "Confirm button in the product downloadables alert")
+        static let downloadableProductAlertCancel = NSLocalizedString("Cancel", comment: "Cancel button in the product downloadables alert")
```

Comment:
Also, for the cancel button how about using "Revert" - it would clarify the intention to revert the change. "Cancel" sounds like it only dismisses the alert.

Reply:
> Nit: Please follow the new convention to use reverse-DNS keys for localized strings (p91TBi-aJl-p2)

Thanks for sharing this, I didn't know about it, I'll update the keys.

> Also, for the cancel button how about using "Revert" - it would clarify the intention to revert the change. "Cancel" sounds like it only dismisses the alert.

I think `Cancel` makes more sense here, we don't update the model until `Yes` is clicked. Also this aligns with the design and keeps both platforms aligned, WDYT?

Reply:
If we follow my [suggestion](https://github.com/woocommerce/woocommerce-ios/pull/11388#discussion_r1422032764) above about toggling the switch only if the user taps the Cancel button, then "Revert" makes more sense to me 😅

Reply:
The keys were updated in aabf47280145695ff9771a090f908577f36e6aaa

Reply:
> If we follow my https://github.com/woocommerce/woocommerce-ios/pull/11388#discussion_r1422032764 above about toggling the switch only if the user taps the Cancel button, then "Revert" makes more sense to me 😅

I followed, but still kept `Cancel` 😅, let me know if it's a blocking point, otherwise personally I prefer to keep both platforms aligned with the design 🙂

Reply:
Please feel free to keep it if you feel strongly about it 👍